app/employee-app/backend/modules/database/db_queries.bal

```diff
@@ -51,16 +51,32 @@ isolated function getCommonEmployeeQuery() returns sql:ParameterizedQuery => `
         hris_employment_type het ON e.employee_employment_type_id = het.employment_type_id
 `;
 
+# Build query to retrieve the distinct employee locations.
+#
+# + employeeStatuses - List of employee statuses to consider
+# + return - Select query for getting distinct locations of the employee table
+isolated function getDistinctEmployeeLocationQuery(string[]? employeeStatuses) returns sql:ParameterizedQuery {
```

Comment:
This is not used in anywhere right? Shall we remove this?

Reply:
We need to get distinct locations of employees for the `leave` app. Will add the resource method for this.

Reply:
Shall we go through the use case offline? We have to clarify the requirements first. Until then let's remove the unwanted functions. PR should not have unneeded functions or half-baked codes.

Reply:
Method removed after offline discussion.